controllers/post-controllers.js

```diff
@@ -0,0 +1,26 @@
+const { 
+    insertComment
+ } = require('../models/comments-models')
+
+function postComment(req, res, next) {
+    const {username} = req.body
+    const {body} = req.body
+    if (!username || !body) {
+        next({status: 400, msg: 'invalid comment object'})
+    }
+    if (typeof username !== 'string' || typeof body !== 'string') {
```

Comment:
If either of these are the wrong data type, PSQL will handle this for you already.